src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,9 @@
+$day-size: 100px;
+$calendar-width: (7 * 100px + 6px + 2 * 10px);
```

Comment:
Use variables to calculate.
100px = $day-size
7 = $week-length
6 = $week-length - 1
10px = ???